### DISCUSSION TITLE
What about Oracle?

### DISCUSSION THREAD

**Discussion by obonilla66** [alfateam/rdb#71] [Q&A] [answered]
What about Oracle?

**Comment by lroal** [comment, marked as answer]
Yes, I did start on it, but took a break because I ran into some small obstacles. But thanks for reminding me and giving me a push. It is absolutely doable.
I will start working on it again late next week.
Merry Christmas!

**Reply by lroal** [reply]
Can i ask for a favour, would you care to star the repository ?

**Comment by lroal** [comment]
I am done with the tests for Oracle now. https://github.com/alfateam/rdb/actions/runs/7627269154
You find everything in this branch https://github.com/alfateam/rdb/tree/oracle/tests .
I will need to write some documentation before releasing it later this week.
For now it only supports TIMESTAMP - not DATE. If DATE it is critical, I can implement it.

**Comment by lroal** [comment]
rdb@3.5.0 with Oracle support is out on npm
